cernopendata/modules/theme/assets/semantic-ui/js/records/FilesBoxApp.js

```diff
@@ -0,0 +1,97 @@
+/*
+ * -*- coding: utf-8 -*-
+ *
+ * This file is part of CERN Open Data Portal.
+ * Copyright (C) 2021 CERN.
+ *
+ * CERN Open Data Portal is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * CERN Open Data Portal is distributed in the hope that it will be
+ * useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with CERN Open Data Portal; if not, write to the
+ * Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+ * MA 02111-1307, USA.
+ *
+ * In applying this license, CERN does not
+ * waive the privileges and immunities granted to it by virtue of its status
+ * as an Intergovernmental Organization or submit itself to any jurisdiction.
+ */
+
+import React, { useEffect, useState } from "react";
+import { Pagination } from "semantic-ui-react";
+
+import { FileTable } from "./components";
+import config from "./config";
+
+const RECORD_FILEPAGE_URL = (pid, page, type = null) =>
+  `/record/${pid}/filepage/${page}?${type ? `type=${type}` : "group=1"}`;
+const ITEMS_PER_PAGE = 5;
```

Comment:
Minor: what do you think about moving `ITEMS_PER_PAGE` and `RECORD_FILEPAGE_URL` to `config.js`? Just in case in the future something changes and there would be a single file to quickly update these kind of things without going into implementation